kernel/util.c

```diff
@@ -6,6 +6,9 @@
 #include <asm/pgtable.h>
 #endif
 #include <linux/printk.h>
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 7, 0)
+#include <linux/sched.h>
+#endif
```

Comment:
```suggestion
#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 11, 0)
#include <linux/sched/task.h>
#else
#include <linux/sched.h>
#endif
```

Would be nicer to make it look like this

Reply:
> Would be nicer to make it look like this

In kernel version 5.7 and later, the definition of `task_struct` is implicitly included via `linux/mm.h`. Should I still explicitly include the corresponding header file in these versions?